django_dbq/management/commands/queue_depth.py

```diff
@@ -7,14 +7,17 @@ class Command(BaseCommand):
     help = "Print the current depth of the given queue"
 
     def add_arguments(self, parser):
-        parser.add_argument("queue_name", nargs="?", default="default", type=str)
+        parser.add_argument("queue_name", nargs="*", default=["default"], type=str)
 
     def handle(self, *args, **options):
-        queue_name = options["queue_name"]
+        queue_names = options["queue_name"]
```

Comment:
Should we rename the argument to `queue_names` too for consistency?

Reply:
so when i run `manage.py queue_depth --help` it outputs the following:

```
usage: manage.py queue_depth [-h] [--version] [-v {0,1,2,3}]
                             [--settings SETTINGS] [--pythonpath PYTHONPATH]
                             [--traceback] [--no-color]
                             [queue_name [queue_name ...]]
```

it would maybe look a little weird if that said `queue_names` instead?

Reply:
Ah ok. Yes I do think it reads better like that, actually.